### PR TITLE
feature: implement vllm benchmark tracing for analysis

### PR DESCRIPTION
**Pull request by rebel-wonsubkim**
### 🚀 Summary of Changes

> *What does this PR do? What feature, fix, or improvement does it bring?*
* import https://github.com/rebellions-sw/fde_bench_ci/tree/dev/profiler feature
* for each request, record start time, end time, first scheduled time to calculate request queue time
* report analysis result of service time & queue time

---

### 📌 Related Issues / Tickets

* Resolves #
* Related to #

---

### ✅ Type of Change

* [ ] 🚀 Release (`release`)
* [x] ✨ Feature (`feature`)
* [ ] 🧠 Model support (`model`)
* [ ] 🧬 Core engine changes (`core`)
* [ ] 🛠 Bug fix (`fix`)
* [ ] ⚙️ Performance improvement (`perf`)
* [ ] 🔁 Refactor or code cleanup (`refactor`)
* [ ] 📄 Documentation (`docs`)
* [ ] ❓ Other (`other`): please describe

---

### 🧪 How to Test

1. Run `...`
2. Verify output: `...`
3. Edge case tested: `...`

---

### 📸 Screenshots / Logs (if applicable)

---

### 📋 Checklist

* [ ] PR title follows Conventional Commits format
* [ ] This PR is linked to an existing issue
* [ ] The test method is described, and the expected result is clearly stated
* [ ] Relevant documentation has been updated (if applicable)

---

### 💬 Notes

---

### CODE REVIEW

File: vllm_rbln/v1/tracing/README.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
+# Perfetto Tracing for vLLM-RBLN
```
Comment (rebel-jinhwan) [inline vllm_rbln/v1/tracing/README.md:1]: it'd be better to move this doc under docs/

File: vllm_rbln/v1/tracing/__init__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+This subpackage applies runtime monkey-patches to vLLM's `EngineCore`,
+`AsyncLLM`, and OpenAI API server so that per-request timestamps
+(arrival, first scheduled, prefill, decode, finish) are emitted as
+Chrome Trace JSON events. Trace lifecycle is controlled via the
+`/v1/trace/start` and `/v1/trace/stop` HTTP endpoints.
+
+Patches are applied automatically when `patches` is imported, which
+is wired into `vllm_rbln.register_ops()`.
+"""
```
Comment (rebel-jinhwan) [inline vllm_rbln/v1/tracing/__init__.py:25]: For consistency with upstream, how about placing under vllm_rbln/ not under vllm_rbln/v1 ? https://github.com/vllm-project/vllm/tree/v0.18.1/vllm/tracing

File: vllm_rbln/v1/tracing/README.md [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+This registers the `vllm-rbln` console script, which auto-loads the tracing
+patches before invoking the vLLM CLI (needed because `vllm bench serve`
+does not load plugins on its own).
+
+## Usage
+
+### 1. Start the server
+
+```bash
+vllm-rbln serve <model> --port 8000 ...
```
Comment (rebel-jinhwan) [inline vllm_rbln/v1/tracing/README.md:32]: is it necessary to use `vllm-rbln` ?

**Comment by rebel-minhopark** [comment]
When do you think this branch will be merged? @rebel-wonsubkim 
I will change shared-ci-configs and fsw-integration test code/suite after this pr is merged.
